ada_feeding/ada_feeding/behaviors/acquisition/compute_food_frame.py

```diff
@@ -139,6 +139,7 @@ def initialise(self):
     @override
     def update(self) -> py_trees.common.Status:
         # Docstring copied from @override
+        self.logger.info(f"{self.name} [{self.__class__.__name__}::update()]")
```

Comment:
Same as above, DebugVisitor should handle node traversal, best not to clog up the screen with INFO messages (which will be the default for running the demo).